pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -61,6 +62,10 @@ const (
 	// is just starting to be reconciled
 	reasonRunning = "Running"
 
+	// reasonBuilding indicates that the reasono for the in progress status is that the TaskRun
+	// is just being built
+	reasonBuilding = "Building"
```

Comment:
what does "buliding" mean in this context? since this whole project started from "knative build", the term Build is pretty overloaded - and i dont think we're actually building anything here :thinking:  is there a more specific term we could use?

Reply:
hrm. We were already using the string "building" as the `reason` for a TaskRun with a running Pod.  This change simply put the string into a variable so that it could be referred to from unit tests without using the literal string in multiple places.

I'd be happy to change this but it doesn't feel related to this PR.  Modifying the TaskRun's Status could also be considered an API change? I'd want to make sure it's isolated in its own commit at least so that we can revert it easily.  I'll make a separate PR for this if we can come up with a better descriptor.  "Executing" might work instead?  "Running" is already taken for a TaskRun that has started up but doesn't yet have a `ConditionSucceeded` status.